dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -30,6 +29,8 @@
  *= require details-polyfill
  */
 
+// TODO: Move into code-studio.scss when it exists.
+@import "/code-studio/assets/video-js/video-js.css";
```

Comment:
Interesting - I hadn't thought of using browserify for this.  It's a [much-requested feature](https://github.com/sass/sass/issues/556) of sass to be able to inline css files but they keep pushing back because it doesn't play nice with their "It's just CSS" philosophy.

I was going to try out [node-sass-css-importer](https://www.npmjs.com/package/node-sass-css-importer) which requires you to be explicit (special syntax) when asking to inline CSS, but that seems okay to me.